app/models/manageiq/providers/redhat/infra_manager/provision_workflow.rb

```diff
@@ -135,4 +135,13 @@ def validate_memory_limit(_field, values, dlg, fld, _value)
       _("%{description} VM Memory is larger than Memory Limit") % {:description => required_description(dlg, fld)}
     end
   end
+
+  def validate_seal_template(_field, values, _dlg, _fld, _value)
+    seal = get_value(values[:seal])
+    return nil unless seal
+
+    if get_source_vm.platform == 'windows'
+      _("Template Sealing is supported only for non-Windows OS.")
```

Comment:
Is there a reason to have "sealing" starting with capital letter?

Reply:
fixed.